research/efficient-hrl/train.py

```diff
@@ -594,7 +594,7 @@ def train_uvf(train_dir,
                              name='post_update_meta_targets')
 
   if debug_summaries:
-    train_.gen_debug_batch_summaries(batch)
+    train_op.gen_debug_batch_summaries(batch)
```

Comment:
train_op is a tensorflow op.  It would not have a gen_debug_batch_summaries method.